rules/rule_defs.go

```diff
@@ -75,9 +76,14 @@ const (
 	ChainDispatchFromHostEndpoint        = ChainNamePrefix + "from-host-endpoint"
 	ChainDispatchToHostEndpointForward   = ChainNamePrefix + "to-hep-forward"
 	ChainDispatchFromHostEndPointForward = ChainNamePrefix + "from-hep-forward"
+	ChainDispatchSetEndPointMark         = ChainNamePrefix + "set-endpoint-mark"
+	ChainDispatchFromEndPointMark        = ChainNamePrefix + "from-endpoint-mark"
 
-	WorkloadToEndpointPfx   = ChainNamePrefix + "tw-"
-	WorkloadFromEndpointPfx = ChainNamePrefix + "fw-"
+	ChainForwardCheck = ChainNamePrefix + "forward-check"
+
+	WorkloadToEndpointPfx      = ChainNamePrefix + "tw-"
+	WorkloadFromEndpointPfx    = ChainNamePrefix + "fw-"
+	WorkloadSetEndPointMarkPfx = ChainNamePrefix + "sepm-"
```

Comment:
Suggest you reduce the length of this to match `tw-` and `fw-`; I know that those are short enough to allow for the longest possible interface name.  Maybe `sm-`?

Reply:
`sm-` is fine.